### PR TITLE
[api] Fix error handling when extracting user info from LDAP.

### PR DESCRIPTION
**Pull request by adrianschroeter**
forward port from 2.6 branch, most of it seems to be changed meanwhile

### CODE REVIEW

**Comment by bgeuken** [comment]
Duplicate of #2360.

File: src/api/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def extract_proxy_user
       unless @http_user
         if ::Configuration.registration == "deny"
           logger.debug("No user found in database, creation disabled")
-          raise AuthenticationRequiredError.new "User '#{login}' does not exist<br>#{errstr}"
+          raise AuthenticationRequiredError.new "User '#{login}' does not exist.<br>"
```
Comment (bgeuken) [inline src/api/app/controllers/application_controller.rb]: Please remove the \<br\> tag.

Comment (adrianschroeter) [inline src/api/app/controllers/application_controller.rb]: done

**Comment by adrianschroeter** [comment]
okay, let's work with the other pull requets ...

**Pull request closed without merging**